### PR TITLE
Lib improvements

### PR DESCRIPTION
**Pull request by josh-padnick**
- Commit changes that should have been committed in https://github.com/gruntwork-io/docs/pull/889
- Minor clean up to Library overview page
- Inspired by https://github.com/gruntwork-io/gruntwork-io.github.io/pull/811

### CODE REVIEW

**Comment by josh-padnick** [comment]
@ebeneliason Would you mind reviewing and approving this (unless you have recommended changes)?

File: docs/library/usage/contributing.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you're thinking of adding a new feature, before starting any work, we recomme
 questions and get feedback from the maintainers and the community before you sink a lot of time into writing (possibly
 the wrong) code. If there is anything you’re unsure about, just ask!
 
-If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below. The key point is that we want to minimize the risk that you invest time in something we ultimately can't use.
+If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below.
```
Comment (MoonMoon1919) [inline docs/library/usage/contributing.md:21]: @josh-padnick Did this autogenerate again for some reason, or did you modify this file manually?

File: docs/library/overview/modules.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
 # What is a module?
 
-**Modules are reusable "infrastructure building blocks" that describe how to deploy and manage a specific piece of infrastructure, such as a VPC, ECS cluster, or Auto Scaling Group.**
+Modules are reusable "infrastructure building blocks" that describe how to deploy and manage a specific piece of infrastructure, such as a VPC, ECS cluster, or Auto Scaling Group.
```
Comment (MoonMoon1919) [inline docs/library/overview/modules.md:3]: Same question here - did you modify this manually or did it auto regenerate for some reason?

File: docs/library/usage/contributing.md (hunk 1 above)
Comment (josh-padnick) [reply docs/library/usage/contributing.md:21]: Sorry about the confusion. I tried to address this in my PR initial comment but I don't think it was clear. When I submitted https://github.com/gruntwork-io/docs/pull/889, I assumed that if I submitted changes to `_docs-sources` folders, the build would auto-generate these files; I didn't realize we were auto-generating these locally and then committing them.

So in this PR, I attempted to commit the generated files.

File: docs/library/overview/modules.md (hunk 2 above)
Comment (josh-padnick) [reply docs/library/overview/modules.md:3]: See https://github.com/gruntwork-io/docs/pull/1010#discussion_r1273901969.

File: docs/library/usage/contributing.md (hunk 1 above)
Comment (MoonMoon1919) [reply docs/library/usage/contributing.md:21]: ahhh okay no worries!

**Pull request merged**